FakebookNotifications/FakebookNotifications.WebApi/Startup.cs

```diff
@@ -38,7 +38,8 @@ public void ConfigureServices(IServiceCollection services)
                 options.AddDefaultPolicy(
                     builder =>
                     {
-                        builder.WithOrigins("")
+                        builder.WithOrigins("http://localhost:4200",
+                            "https://fakebook.revaturelabs.com")
```

Comment:
it's not useful to add this second line, because the only way a request could get to this API from a page hosted by fakebook.revaturelabs.com is if the API is itself also on fakebook.revaturelabs.com, so it's the same origin.

but it doesn't hurt anything to leave it there